terraform/deployments/variables/test/common.tfvars

```diff
@@ -3,7 +3,7 @@ cluster_infrastructure_state_bucket = "govuk-terraform-test"
 
 cluster_version               = 1.21
 cluster_log_retention_in_days = 7
-workers_default_capacity_type = "SPOT"
+# workers_default_capacity_type = "SPOT"
```

Comment:
Accidental commit? We probably want to continue to use spot instances by default for test clusters.